Source/Core/Core/Core.cpp

```diff
@@ -414,6 +414,9 @@ static void FifoPlayerThread(const std::optional<std::string>& savestate_path,
 // See the BootManager.cpp file description for a complete call schedule.
 static void EmuThread(std::unique_ptr<BootParameters> boot, WindowSystemInfo wsi)
 {
+  // Clear messages printed before the window opened
```

Comment:
This should probably explain why, rather than what you're doing. It's easy enough to see that clearing messages is being performed because the name of the function is literally `ClearMessages()`

Reply:
Because they printed before the window opened (and they are outdated)?